cql3/statements/modification_statement.cc

```diff
@@ -71,14 +72,20 @@ modification_statement_timeout(const schema& s) {
     }
 }
 
-modification_statement::modification_statement(statement_type type_, uint32_t bound_terms, schema_ptr schema_, std::unique_ptr<attributes> attrs_, uint64_t* cql_stats_counter_ptr)
+modification_statement::modification_statement(statement_type type_,
+                                               uint32_t bound_terms,
+                                               schema_ptr schema_,
+                                               std::unique_ptr<attributes> attrs_,
+                                               uint64_t* cql_stats_counter_ptr,
+                                               uint64_t* cql_system_stats_counter_ptr)
```

Comment:
it's not instantly obvious what is "new" and what is just a cosmetic change (shortening LoCs). Please put the cosmetic change into a separate commit.

Reply:
Broken lines should be indented just with a double indent (2x4 spaces).